components/symbl_ai/actions/submit-video-url/post-video-url.js

```diff
@@ -0,0 +1,52 @@
+import axios from "axios";
+
+export default {
+  key: "symbl_ai-post-video-url",
+  name: "Submit Video URL",
+  description: "Submit a Video by providing the URL for processing",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symbl_ai: {
+      type: "app",
+      app: "symbl_ai",
+      label: "",
+      description: "",
+    },
```

Comment:
Then you can repace this with
```suggestion
    symblAIApp,
```